docs/reference/glossary.md

```diff
@@ -328,7 +328,7 @@ You can have one server-worker connected to your [deployment](#deploying), or do
 
 ### Scene
 
-In the GDK, a Unity Scene stores GameObjects that you might use to [represent your SpatialOS entities]({{urlRoot}}/reference/workflows/monobehaviour/linking/spatialos-entities). Each Scene can interact with multiple Unity ECS worlds.
+In the GDK, a Unity Scene stores GameObjects that you might use to represent your SpatialOS entities. Each Scene can interact with multiple Unity ECS worlds.
```

Comment:
do we need to capitalize "worlds" or not?

Reply:
I don't know - I'm tempted to leave it as is

Reply:
checked the glossary, we don't capitalize worlds